Sprint-2/errors/1.js

```diff
@@ -4,10 +4,11 @@
 // Try playing computer with the example to work out what is going on
 
 function convertToPercentage(decimalNumber) {
-  const decimalNumber = 0.5;
+  // const decimalNumber = 0.5;v => thisline cause error
   const percentage = `${decimalNumber * 100}%`;
 
   return percentage;
 }
+// the decimalNumber has already been declared its constant cant except
 
 console.log(decimalNumber);
```

Comment:
When you try executing this script, does it output 50%?

Reply:
Yes , when you execute  console.log(convertToPercentage(0.5)) the output is  50%?